### PR TITLE
Fix bugs

### LINKED ISSUE #176: [PE-D][Tester C] Add book command able to parse invalid name if multiple name tags are provided
user will not get an error message if such an input is provided

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: xav168/ped#2

### LINKED ISSUE #177: [PE-D][Tester A] BookFace app and User Guide does not align
**Describe the bug**

I can give user tags as displayed in the display box but it is not mentioned in the User Guide.

**Screenshots**

*BookFace app:*

*User Guide:*

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: LimAiLin/ped#1

### LINKED ISSUE #181: [PE-D][Tester C] Book with same name but with different capitalization considered as different books
Not sure if this is a bug or its intended just putting it  out here

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: xav168/ped#3

### PR DESCRIPTION
**Pull request by parth-io**
Fixes #176 and fixes #181 
Fixes #177

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -334,22 +334,22 @@ If your changes to the data file makes its format invalid, BookFace will discard
...
-| **List loans**  | `list loans`                                                                                              |
-| **List overdue**| `list overdue`                                                                                            |
-| **Clear**       | `clear all`                                                                                               |
-| **Exit**        | `exit`                                                                                                    |
+| Action          | Format, Examples                                                                                                              |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------|
+| **Add book**    | `add book t/TITLE a/AUTHOR` <br> E.g: `add book t/James and The Giant Peach  a/Roald Dahl`                                    |
+| **Add user**    | `add user n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]...` <br> E.g: `add user n/John Doe p/91234567 t/friend e/johndoe@outlook.com` |
```
Comment (TYKCodes) [inline docs/UserGuide.md:340]: I think the example command given does not match the format (`Tag` should be after `Email`)

**Review by TYKCodes — CHANGES_REQUESTED** [review]
Other than the command example format, everything else looks good to me!

File: docs/UserGuide.md (hunk 1 above)
Comment (parth-io) [reply docs/UserGuide.md]: Fixed

**Review by TYKCodes — APPROVED** [review]
Looks good to me! Thank you for the bug fixes!

**Pull request merged**